joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/Defines.scala

```diff
@@ -0,0 +1,5 @@
+package io.joern.csharpsrc2cpg.astcreation
+
+object Defines {
+  val AwaitOperator = "<operator>.await"
```

Comment:
This should go under `io.joern.csharpsrc2cpg.CSharpOperators` (under `Constants.scala`)